python/pyspark/ml/classification.py

```diff
@@ -656,6 +657,13 @@ class NaiveBayes(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol, H
                  HasRawPredictionCol):
     """
     Naive Bayes Classifiers.
+    It supports both Multinomial NB
+    (`http://nlp.stanford.edu/IR-book/html/htmledition/naive-bayes-text-classification-1.html`)
```

Comment:
It will make this link in italics, I just refer [other links at classification.py](https://github.com/apache/spark/blob/master/python/pyspark/ml/classification.py#L246).